### ISSUE TITLE
Error cannot connect to web.whatsapp.com

### ISSUE THREAD

**Issue by kingrootid** [orkestral/venom#1254]
```info:     [sessionName] Initializing browser wss...
info:     [sessionName] Browser successfully opened
info:     [sessionName] checking headless...
info:     [sessionName] headless option is active, browser hidden
info:     [sessionName] Checking page...
info:     [sessionName] Error accessing the page: "https://web.whatsapp.com"
Error when trying to access the page: "https://web.whatsapp.com"```
```
my script 
```
const venom = require("venom-bot");

venom
  .create(
    //session
    "sessionName", //Pass the name of the client you want to start the bot
    //catchQR
    asciiQR => {
      console.log("Terminal qrcode: ");
    },
    // statusFind
    (statusSession, session) => {
      console.log("Status Session: ", statusSession); //return isLogged || notLogged || browserClose || qrReadSuccess || qrReadFail || autocloseCalled || desconnectedMobile || deleteToken || chatsAvailable || deviceNotConnected || serverWssNotConnected || noOpenBrowser
      //Create session wss return "serverClose" case server for close
      console.log("Session name: ", session);
    },
    // options
    {
      multidevice: true, // enabled multidevice.(default: true)
      folderNameToken: "tokens", //folder name when saving tokens
      mkdirFolderToken: "", //folder directory tokens, just inside the venom folder, example:  { mkdirFolderToken: '/node_modules', } //will save the tokens folder in the node_modules directory
      headless: true, // Headless chrome
      devtools: false, // Open devtools by default
      useChrome: true, // If false will use Chromium instance
      debug: false, // Opens a debug session
      logQR: true, // Logs QR automatically in terminal
      browserWS: "", // If u want to use browserWSEndpoint
      browserArgs: ["--no-sandbox"], //Original parameters  ---Parameters to be added into the chrome browser instance
      puppeteerOptions: {}, // Will be passed to puppeteer.launch
      disableSpins: true, // Will disable Spinnies animation, useful for containers (docker) for a better log
      disableWelcome: true, // Will disable the welcoming message which appears in the beginning
      updatesLog: true, // Logs info updates automatically in terminal
      autoClose: 60000, // Automatically closes the venom-bot only when scanning the QR code (default 60 seconds, if you want to turn it off, assign 0 or false)
      createPathFileToken: true, //creates a folder when inserting an object in the client's browser, to work it is necessary to pass the parameters in the function create browserSessionToken
    }
  )
  .then(client => start(client))
  .catch(erro => {
    console.log(erro);
  });

function start(client) {
  client.onMessage(message => {
    if (message.body === "Hi" && message.isGroupMsg === false) {
      client
        .sendText(message.from, "Welcome Venom 🕷")
        .then(result => {
          console.log("Result: ", result); //return object success
        })
        .catch(erro => {
          console.error("Error when sending: ", erro); //return object error
        });
    }
  });
}
```

after i scan qr and start again the code

**Comment by antirek** [comment]
ping web.whatsapp.com, check internet connection
and upgrade to latest venom version

**Comment by kingrootid** [comment]
solve this issue, but i had another problem 
not save a tokens after scan the qr 

in my option client
```
folderNameToken: "tokens", //folder name when saving tokens
      mkdirFolderToken: "", //folder directory tokens, just inside the venom folder, example:  { mkdirFolderToken: '/node_modules', } //will save the tokens folder in the node_modules directory
      headless: true, // Headless chrome
      devtools: false, // Open devtools by default
      useChrome: true, // If false will use Chromium instance
      debug: false, // Opens a debug session
      logQR: true, // Logs QR automatically in terminal
      browserArgs: ["--no-sandbox"], // Parameters to be added into the chrome browser instance
      disableSpins: true, // Will disable Spinnies animation, useful for containers (docker) for a better log
      disableWelcome: true, // Will disable the welcoming message which appears in the beginning
      updatesLog: true, // Logs info updates automatically in terminal
      autoClose: 60000, // Automatically closes the venom-bot only when scanning the QR code (default 60 seconds, if you want to turn it off, assign 0 or false)
```

**Comment by iamedusm** [comment]
I have the same problem in the version 4.0.0

**Comment by MiniFris** [comment]
Same problem
Version 4.0.0

**Comment by jccalbuquerque** [comment]
I have the same problem.
I updated to version 3.2.0 and the error persists.
The qrcode is generated and read on the smartphone.
But the token is not generated and the status becomes QRCODE...

**Comment by jccalbuquerque** [comment]
I updated to version 4.0.0 and it worked 100% thanks.

**Comment by fernandopraise** [comment]
info:     [Atend_] Initializing browser wss...
info:     [Atend_] Browser successfully opened
info:     [Atend_] checking headless...
info:     [Atend_] headless option is active, browser hidden
info:     [Atend_] Checking page...
info:     [Atend_] Error accessing the page: "https://web.whatsapp.com"
(node:2796) UnhandledPromiseRejectionWarning: Error when trying to access the page: "https://web.whatsapp.com"
(node:2796) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 1)
(node:2796) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.

**Comment by vicfelipe** [comment]
same here

**Comment by AlexsandroZanella2** [comment]
> ### Vocês antes de instalar o venom estão fazendo a limpeza do cache do npm?
> ### Limpado a pasta temporária do s.o?
> Testei varias vezes aqui, desconectei e conectei e não acontece isso.
> 
> Versão testadas 4.0 e 3.12

I tried it, and the problem remains;

**Comment by AlexsandroZanella2** [comment]
For some reason, after I updated puppeteer, run npm cache clear, and changed the version to 3.1.12, it started working again;

**Comment by fernandopraise** [comment]
venom 4.0.0
  "puppeteer": "^5.5.0",
        "puppeteer-extra": "^3.1.18",
        "puppeteer-extra-plugin-stealth": "^2.9.0",

**Comment by vicfelipe** [comment]
Problem solved, but needs changes to fix.

When start the server for the first time and qr code was read.
Everything will works fine.


![image](https://user-images.githubusercontent.com/12769395/140797571-8883c78a-1589-4d18-b39f-5c24cf6d32e9.png)

Also when you restart the server, the error appears.

![image](https://user-images.githubusercontent.com/12769395/140797811-f4d428ac-493f-4048-b8bc-394ef2ca32fe.png)

How to fix:

Removing folder "tokens" and restart server.

![image](https://user-images.githubusercontent.com/12769395/140797937-4abbdc7b-81a4-40ba-9dc2-edaaf3759175.png)

A possible solution to be implemented is to remove the folder when the server is shutting down.

**Comment by fernandopraise** [comment]
× [instance: Atend_]: Page Closed
× [instance: Atend_]: Phone not connected
Phone not connected
(node:2876) UnhandledPromiseRejectionWarning: Error: Protocol error (Runtime.callFunctionOn): Target closed.
    at C:\Users\Sistemas\node_modules\puppeteer\lib\cjs\puppeteer\common\Connection.js:208:63
    at new Promise (<anonymous>)
    at CDPSession.send (C:\Users\Sistemas\node_modules\puppeteer\lib\cjs\puppeteer\common\Connection.js:207:16)
    at ExecutionContext._evaluateInternal (C:\Users\Sistemas\node_modules\puppeteer\lib\cjs\puppeteer\common\ExecutionContext.js:200:50)
    at ExecutionContext.evaluateHandle (C:\Users\Sistemas\node_modules\puppeteer\lib\cjs\puppeteer\common\ExecutionContext.js:151:21)
    at WaitTask.rerun (C:\Users\Sistemas\node_modules\puppeteer\lib\cjs\puppeteer\common\DOMWorld.js:528:37)
    at processTicksAndRejections (internal/process/task_queues.js:97:5)
(node:2876) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 1)
(node:2876) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.
(node:2876) UnhandledPromiseRejectionWarning: Error: Protocol error (Runtime.callFunctionOn): Target closed.
    at C:\Users\Sistemas\node_modules\puppeteer\lib\cjs\puppeteer\common\Connection.js:208:63
    at new Promise (<anonymous>)
    at CDPSession.send (C:\Users\Sistemas\node_modules\puppeteer\lib\cjs\puppeteer\common\Connection.js:207:16)
    at ExecutionContext._evaluateInternal (C:\Users\Sistemas\node_modules\puppeteer\lib\cjs\puppeteer\common\ExecutionContext.js:200:50)
    at ExecutionContext.evaluateHandle (C:\Users\Sistemas\node_modules\puppeteer\lib\cjs\puppeteer\common\ExecutionContext.js:151:21)
    at WaitTask.rerun (C:\Users\Sistemas\node_modules\puppeteer\lib\cjs\puppeteer\common\DOMWorld.js:528:37)
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:97:5)
(node:2876) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 2)

**Comment by orkestral** [comment]
fixed in version 4.0.1
